build/lib/release.sh

```diff
@@ -377,7 +401,7 @@ EOF
           echo "COPY nsswitch.conf /etc/" >> "${docker_file_path}"
         fi
 
-        "${DOCKER[@]}" build ${docker_build_opts} -q -t "${docker_image_tag}" "${docker_build_path}" >/dev/null
+        "${DOCKER[@]}" build "${docker_build_opts}" -q -t "${docker_image_tag}" "${docker_build_path}" >/dev/null
```

Comment:
doesn't this expand to `""` for `docker_build_opts=` instead of null?
I think if we switch from an array we want `${docker_build_opts:+"${docker_build_opts}"}`

Reply:
```
$ (set -x; foo=; echo ${foo})
+ foo=
+ echo

$ (set -x; foo=; echo "${foo}")
+ foo=
+ echo ''

$ (set -x; foo=; echo ${foo:+"${foo}"})
+ foo=
+ echo
```

Reply:
You are right. Updated.